themes/icons/send.svg

```diff
@@ -1,4 +1,4 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" fill="none">
-  <path d="M1.00053 1.00103L14.9995 8.00537L1.00053 14.999L3.69265 8.00537L1.00053 1.00103Z" stroke="#555C65" stroke-width="2" stroke-linejoin="round"/>
```

Comment:
These hard coded attributes were overridden by our CSS, so they had no effect anyway

Reply:
Contrary to HTML, in SVG attributes have lower priority than classes: https://stackoverflow.com/questions/47088409/svg-attributes-beaten-by-cssstyle-in-priority